src/transform/parameters.ts

```diff
@@ -19,11 +19,16 @@ export function transformParametersArray(
 
   // sort into map
   let mappedParams: Record<string, Record<string, ParameterObject>> = {};
-  for (const paramObj of parameters as any[]) {
```

Comment:
I factored out the cast to `as any[]` and made some changes below to satisfy the type violations caused by that.